### DISCUSSION TITLE
Is this still being worked with?

### DISCUSSION THREAD

**Discussion by katgro99** [ramon/fvtt-thewitcher#4] [General]
Hi!
First of all, mega cool that you took it upon yourself to start a project like this - really appreciate it!
I just wanna ask if you're still working on this or if the project is on hold, or completely abandoned?
It's not ment as a way of rushing you or anything just curious if the project is still ongoing.
Thx again for starting this project at all!
- Kat

**Comment by ramon** [comment]
Hi, Kat, is on hold, im without time, but a have plans to finish at least the character sheet and fully support for the all kind of itens.
